copyright

```diff
@@ -1449,6 +1507,23 @@ Copyright: Saugia <https://github.com/Saugia>
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (under the same license) and Lia Gerty (under the same licence).
 
+Files:
+ images/outfit/nanobot?defense*
+Copyright: Saugia <https://github.com/Saugia>
+Comment: Derived from work by Griffin Schutte (theronepic@gmail.com) (under same licence).
+License: CC-BY-SA-4.0
+
+Files:
+ images/outfit/liquid?sodium*
+Copyright: Saugia <https://github.com/Saugia>
+Comment: Derived from works by NomadicVolcano (public domain).
+License: CC-BY-SA-4.0
+
+Files:
+ sounds/remnant afterburner.wav
+License: public-domain
+Comment: Based on public domain sounds taken from freesound.org, edit done by Saugia.
+
```

Comment:
```suggestion

Files:
 images/outfit/fuel?stack*
 images/outfit/korath?gel?gun*
 images/outfit/korath?gel?turret*
 images/outfit/liquid?sodium*
Copyright: Saugia <https://github.com/Saugia>
Comment: Derived from works by NomadicVolcano (public domain).
License: CC-BY-SA-4.0

```
Didn't look like you had these yet.

Reply:
Gel Gun and Fuel Stack aren't in this PR